src/array_api_stubs/_2023_12/searching_functions.py

```diff
@@ -107,17 +107,11 @@ def searchsorted(
     side: Literal['left', 'right']
         argument controlling which index is returned if a value lands exactly on an edge.
 
-        Let ``x`` be an array of rank ``N`` where ``v`` is an individual element given by ``v = x2[n,m,...,j]``.
+        Let ``x2`` be an array of rank ``N`` where ``v`` is an individual element given by ``v = x2[n,m,...,j]``.
```

Comment:
I don't think `N` is defined or referred to before or again, so it might be fine to just omit that.
Similarly for `n`, `m`, and `j`. To be precise, it will help to define one of these and refer to it later, so:
```suggestion
        Let ``v`` be an element of ``x2`` given by ``v = x2[j]``, where ``j`` refers to a valid scalar or tuple index.
```
(If you feel it's necessary to state that scalar index would be for a 1-D array and tuple for multidimensional, that's fine, too.)